includes/library/zencart/listingBox/src/formatters/TabularProduct.php

```diff
@@ -121,6 +122,12 @@ protected function getRowLayoutMap($item, $prodLink)
                     zen_trunc_string(zen_clean_html(stripslashes($prodDesc)), PRODUCT_LIST_DESCRIPTION) . '</div>';
                 return array('value' => $lc_text, 'col_params' => '');
             },
+            'PRODUCT_LIST_DESCRIPTION' => function () use ($item, $prodLink) {
+                $prodDesc = zen_get_products_description($item ['products_id'], $_SESSION ['languages_id']);
+                $lc_text = '<div class="listingDescription">' .
+                    zen_trunc_string(zen_clean_html(stripslashes($prodDesc)), PRODUCT_LIST_DESCRIPTION) . '</div>';
+                return array('value' => $lc_text, 'col_params' => 'colspan="'. sizeof($this->columnList) . '"');
+            },
```

Comment:
Hmmm ... might need a different approach than `colspan` for `colspan="'. sizeof($this->columnList) . '"`, since `$this->columnList` doesn't exist in this scope.